src/features/Chapters/Chapters.tsx

```diff
@@ -1,29 +1,13 @@
-import React, { useContext, useEffect } from 'react';
+import React, { useContext } from 'react';
 import { P } from 'components/P';
-import { useChapterCreate, useChaptersByCoverId } from 'apis/useChapters';
```

Comment:
I think it would be better to put API feature component next to it, instead of in the general apis folder, because in case of API extension or its revision, it is better to immediately see those files with which you will have to work, instead of opening the apis folder where there will be a lot of files.